code/game/gamemodes/cult/runes.dm

```diff
@@ -941,7 +941,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 	notify_ghosts("Manifest rune created in [get_area(src)].", ghost_sound = 'sound/effects/ghost2.ogg', source = src)
 	var/list/ghosts_on_rune = list()
 	for(var/mob/dead/observer/O in T)
-		if(O.client && !iscultist(O) && !jobban_isbanned(O, ROLE_CULTIST) && !O.has_enabled_antagHUD && !QDELETED(src) && !QDELETED(O))
+		if(O.client && !iscultist(O) && !jobban_isbanned(O, ROLE_CULTIST) && !O.has_enabled_antagHUD && !QDELETED(src) && !QDELETED(O) && !HAS_TRAIT_FROM(O.mind.current, SCRYING, CULT_TRAIT))
```

Comment:
I think this should be applied to this trait in general. The only other source of this is the wizard scrying orb, which in the rare case it happens, probably also shouldnt be able to become a cult ghost.
```suggestion
		if(O.client && !iscultist(O) && !jobban_isbanned(O, ROLE_CULTIST) && !O.has_enabled_antagHUD && !QDELETED(src) && !QDELETED(O) && !HAS_TRAIT(O.mind.current, SCRYING))
```